### PR TITLE
Improvement: make it clear that the synced objects are the ones on the device

### PR DESCRIPTION
**Pull request by omnajjar**

### CODE REVIEW

**Comment by omnajjar** [comment]
Will pass it to the team for a review, and once ll good I'll unmark it as a draft!

File: content/refguide/synchronize.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The behavior of this mode can be configured through [synchronization configurati
 ![Synchronize](attachments/client-activities/synchronize-unsynchronized-objects.png)
 {{% /image_container %}}
 
-With the **Unsychronized objects** mode, all objects with committed changes are synchronized. The synchronization is bi-directional, meaning both the server database and the local database are updated for these objects. For more information, see the [Synchronization Behavior](#synchronization-behavior) section below.
+With the **Unsychronized objects** mode, all objects on the device with committed changes are synchronized. The synchronization is bi-directional, meaning both the server database and the local database are updated for these objects. For more information, see the [Synchronization Behavior](#synchronization-behavior) section below.
```
Comment (ekrem-aktas) [inline content/refguide/synchronize.md:35]: We have offline PWA profiles as well, doesn't mentioning the "device" sound like it is valid for only native apps?

Comment (omnajjar) [reply content/refguide/synchronize.md:35]: Yeah indeed, it won't be interpreted to a browser as well! .... what do you think then of: 
`... all objects with committed changes locally are synchronized...`

Comment (ekrem-aktas) [reply content/refguide/synchronize.md:35]: How about something like this: "... all objects that have been committed to the offline database are synchronized"?

Comment (omnajjar) [reply content/refguide/synchronize.md:35]: Yes, that's more specific and inclusive! 👌🏼   ... will update it!

Comment (omnajjar) [reply content/refguide/synchronize.md]: Updated! ✔️

**Review by ekrem-aktas — APPROVED** [review]
LGTM

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**